apps/teams-test-app/src/components/Links.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ReactElement } from 'react';
+
+import { ApiWithTextInput } from './utils';
+
+const MoveToLink = (): ReactElement =>
+  ApiWithTextInput<string>({
+    name: 'moveToLink',
+    title: 'Move to Link',
+    onClick: {
+      validateInput: input => {
+        if (typeof input !== 'string') {
+          throw new Error('Input should be a string');
```

Comment:
Do we mean to say input is a url string? I think string seems too generic and people might enter any string.

Reply:
oh, just to be more clear, I meant to say if it might be more clear if it says below :)
 throw new Error('Input should be a url string');

Reply:
I agree and that behavious we get it from URL construct itself. It throws an exception with invalid URL message if it isn't able to form a URL. For eg. "Failed to construct 'URL': Invalid URL"